app.js

```diff
@@ -12,10 +14,38 @@ app.use(bodyParser.json({
   }
 }));
 
+waitForDatabase(initScheduledJobs);
+
 app.get('/', function (_, res) {
   res.send('Hello from job-controller');
 });
 
+app.post('/delta-scheduledJob', function(req, res) {
+  const scheduledJobToAdd = new Delta(req.body).getInsertsFor('http://www.w3.org/1999/02/22-rdf-syntax-ns#type', 'http://vocab.deri.ie/cogs#ScheduledJob');
```

Comment:
Extra comment about: https://github.com/lblod/job-controller-service/pull/2#discussion_r705449625
Actually if you check for this one only http://schema.org/repeatFrequency, it will probably work in 99% of the times